File: Common/Models/AccountModel.cs
using Common.Models.Base;

namespace Common.Models;

public class AccountModel : BaseModel
{
    public string AccountCode { get; set; }

    public string OwnerName { get; set; }

    public long AgentFiscalCode { get; set; }

    public string BankName { get; set; }

    public string BankCode { get; set; }

    public long AmountOfOutcomingOrders { get; set; }

    public long AmountOfIncomingOrders { get; set; }
}

File: ui/src/entities/account/model/types/AccountModel.ts
import {ModelConcurrencyTokenInterface} from "../../../types";

export interface AccountModel extends ModelConcurrencyTokenInterface {
    id: number,
    accountCode: string,
    "ownerName": string,
    agentFiscalCode: number,
    bankName: string,
    bankCode: string,
    amountOfOutcomingOrders: number,
    amountOfIncomingOrders: number
}